src/util/BaseLoginController.js

```diff
@@ -92,8 +94,12 @@ define(['okta', 'vendor/lib/q'], function (Okta, Q) {
     toggleButtonState: function (state) {
       var button = this.$el.find('.button');
       button.toggleClass('link-button-disabled', state);
-    }
+    },
 
+     // Event triggered after a page is rendered along with the classname to identify the page
+    postRenderAnimation: function() {
+        this.trigger('pageRendered', {page: this.className});
```

Comment:
nits: Comment should probably be inside the function since it's talking about the trigger

Also, it looks like the spacing here is not correct (should not be indented as much)